### ISSUE TITLE
Accept Tuple action spaces

### ISSUE THREAD

**Issue by aowen87** [LLNL/Abmarl#506]
I'm trying to use a Tuple action space with Abmarl to support mixed action types, but I'm getting the following error:
```
AssertionError: The action space must be None, a gym Space, or a dict of gym Spaces.
```

Can Abmarl be updated to accept Tuple action spaces?

**Comment by aowen87** [comment]
FWIW, I'm actually using a subclass of `Tuple` that I'm calling `FlatteningTuple`. The only difference is that its samples are flattened into a single numpy array rather than being a tuple of numpy array samples.

**Comment by rusu24edward** [comment]
I believe the underlying issue is that ppo and friends is using gymnasium, whereas I haven't updated Abmarl to use gymnasium yet, still using gym. So FlatteningTuple is a subclass of gymnasium.spaces.Tuple, but abmarl checks if its a subclass of gym.spaces.Tuple. I believe (and probably other bugs) will be resolved with Abmarl's support for gymnasium.

This should be resolved with #391, I will leave this open until I merge that and can confirm that it fixes the problem.
